### PR TITLE
Add lua applet to aid tuning of IM_STB_COL curve in heli. 

### PR DESCRIPTION
**Pull request by Gone4Dirt**
This PR adds an applet and readme for a simple tool that is useful for tuning the IM_STB_COL_2 and IM_STB_COL_3 parameters in a more intuitive way.  More details about the script can be found in the read me.

Two lua bindings have also been added to access RC functions: norm_input_ignore_trim() and get_reverse()

Tested in SITL.

### CODE REVIEW

File: libraries/AP_Scripting/applets/Heli_IM_Col_Tune.lua [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+-- at the midstick.
+
+-- Tested and working as of 21st Aug 2020 (Copter Dev)
+
+
+function update()
+    local rcin_50 = col50_val_ch:norm_input_ignore_trim()
+    local rcin_grad = col_grad_ch:norm_input_ignore_trim()
+
+    -- Ensure that the channel hasn't been reversed
```
Comment (IamPete1) [inline libraries/AP_Scripting/applets/Heli_IM_Col_Tune.lua:15]: Why?

File: libraries/AP_Scripting/applets/Heli_IM_Col_Tune.lua [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    end
+
+    -- Enforce that IM_STB_COL_4 > IM_STB_COL_3
+    im_100_pct = get_im_val('IM_STB_COL_4',false)
+    if im_100_pct <= im_60_pct then
+        -- Correct within parameter limits
+        im_100_pct = min_max(im_60_pct + 1)
+
+        -- Set correct value to prevent pre-arm warnings
+        if not(param:set('IM_STB_COL_4',im_100_pct)) then
```
Comment (IamPete1) [inline libraries/AP_Scripting/applets/Heli_IM_Col_Tune.lua:64]: only setting the params will not be persistent you would need to set_and_save

Comment (WickedShell) [reply libraries/AP_Scripting/applets/Heli_IM_Col_Tune.lua:64]: Arguably you would want to set it normally, and have a separate switch/interaction to set and save, but as long as people back it up first it's probably okay?

File: libraries/AP_Scripting/applets/Heli_IM_Col_Tune.lua (hunk 1 above)
Comment (Gone4Dirt) [reply libraries/AP_Scripting/applets/Heli_IM_Col_Tune.lua:15]: Fair, probably best to leave that option to the user.  Removed

File: libraries/AP_Scripting/applets/Heli_IM_Col_Tune.lua (hunk 2 above)
Comment (Gone4Dirt) [reply libraries/AP_Scripting/applets/Heli_IM_Col_Tune.lua:64]: I have added a switch that saves the params and exits the script

**Review by tridge — APPROVED** [review]

**Pull request merged**